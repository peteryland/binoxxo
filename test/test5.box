2oo3o2
o6o
4o3x
x_o3o
7x_o
_xx
6oo_o
_o6x
5o
x
