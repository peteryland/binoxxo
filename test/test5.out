__OO___O__
O______O__
____O___X_
X_O___O___
_______X_O
_XX_______
______OO_O
_O______X_
_____O____
X_________

OXOOXOXOXX
OXOXXOXOOX
XOXOOXOXXO
XOOXOXOOXX
OXOXXOXXOO
OXXOOXOXOX
XOXOXXOOXO
XOOXOOXOXX
OXXOXOXXOO
XOXXOXOXOO

